pages/common/pio-upgrade.md

```diff
@@ -0,0 +1,12 @@
+# pio upgrade
+
+> Update PlatformIO to the latest version.
```

Comment:
```suggestion
> Update PlatformIO to the latest stable/development version.
```

Reply:
But isn't development unstable?

Reply:
Yeah, I'm trying to say "Update PlatformIO to the latest stable or the latest development version," but it doesn't matter.  Since you're confused, there's a good chance many others may be as well, so let's not apply this suggestion.